components/alert_banner.tsx

```diff
@@ -9,7 +9,7 @@ import './alert_banner.scss';
 type Props = {
     mode: 'danger' | 'warning' | 'info';
     title?: React.ReactNode;
-    message: React.ReactNode;
+    message?: React.ReactNode;
```

Comment:
why did you make that optional? IMO a alert banner should always have a message! :D